core-bundle/src/Cron/PurgeOptInTokensCron.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+declare(strict_types=1);
+
```

Comment:
The CS fixer currently does not automatically insert the copyright comment here - but only for the `Cron` sub namespaces for some reason.